src/projects/intelligent-user-interfaces-for-virtual-healthcare.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: layouts/base.njk
+title: "Intelligent User Interfaces for Virtual Healthcare"
+description: >-
+  *Intelligent User Interfaces for Virtual Healthcare* aims to reduce barriers to accessing healthcare through inclusive guidelines for (virtual or in-person) healthcare and an approach to personalization in these settings.
+
+  ![Intelligent User Interfaces for Virtual Healthcare logo](/media/inclusive-virtual-healthcare.png)

```

Comment:
Is this really something that needs alt text and isn't just presentational? Should it really be part of the link? We may need to revisit the design of the projects page in general if changes are needed for these questions. If so, it should probably be filed as a separate issue.

Reply:
Addressing #212 will remove the need for these images.

So no need to file a new issue unless we want a way to mark images as presentational using the Rich Editor.